Command/RunCommand.php

```diff
@@ -375,13 +381,24 @@ private function startJob(Job $job)
      *
      * In such an error condition, these jobs are cleaned-up on restart of this command.
      */
-    private function cleanUpStaleJobs($workerName)
+    private function cleanUpStaleJobs($workerName, $restrictedQueues)
```

Comment:
I think we do not need this here. Each worker name must be unique, so the previous query should work fine, no?